### PR TITLE
IA chart bump for documents and restart AAT

### PR DESCRIPTION
**Pull request by makowalski**
### JIRA link (if applicable) ###
N/A

### Change description ###
IA chart bump for documents and restart AAT

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/aat/common/ia/case-api.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ia/case-api:prod-dbff1f2f
```
Comment (jkumar-moj) [inline k8s/aat/common/ia/case-api.yaml]: Don't we need version bump here?

File: k8s/aat/common/ia/case-notifications-api.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ spec:
       image: hmctspublic.azurecr.io/ia/case-notifications-api:prod-d3199230
       ingressHost: ia-case-notifications-api-aat.service.core-compute-aat.internal
```
Comment (jkumar-moj) [inline k8s/aat/common/ia/case-notifications-api.yaml]: Don't we need version bump here?

File: k8s/aat/common/ia/case-api.yaml (hunk 1 above)
Comment (makowalski) [reply k8s/aat/common/ia/case-api.yaml]: nope, version is bumped only for documents api, case-api needs only restart

existing version 0.0.5 matches https://github.com/hmcts/ia-case-api/blob/master/charts/ia-case-api/Chart.yaml#L4
version in application

Comment (jkumar-moj) [reply k8s/aat/common/ia/case-api.yaml]: Thanks Marcin..

**Pull request merged**